src/components/Footer.jsx

```diff
@@ -0,0 +1,89 @@
+import { Link } from 'react-router-dom';
+
+const Footer = () => {
+  return (
+    <footer className="bg-white w-full absolute bottom-0">
+      <div className="px-4 py-12 mx-auto overflow-hidden max-w-7xl sm:px-6 lg:px-8">
+        <nav
+          className="flex flex-wrap justify-center -mx-5 -my-2"
+          aria-label="Footer"
+        >
+          <div className="px-5 py-2">
+            <a href="#" className="text-sm text-gray-500 hover:text-blue-600">
+              Follow us
+            </a>
+          </div>
+        </nav>
+        <div className="flex justify-center mt-4 space-x-6">
+          <span className="inline-flex justify-center w-full gap-3 m-auto md:justify-start md:w-auto">
+            <Link
```

Comment:
Advice:

You have 4 different Links that have very similar content. So, you can create an array with the links information, for example:

const SOCIAL_NETWORKS = [
{
    name: 'github',
    Label: 'Github'
    url:''
},
{
    name: 'twitter',
    Label: 'twitter',
    url:''
},
{
    name: 'instagram',
    Label: 'Instagram',
    url:''
},
{
    name: 'linkedin',
    Label: 'Linkedin',
    url:''
},
]

And render the Links container by iterating the array:

{ SOCIAL_NETWORKS.map(socialNetwork => {
return (
    <Link
              to={socialNetwork.url}
              target="_blank"
              className="w-6 h-6 transition fill-black hover:text-blue-500"
            >
              <span className="sr-only">{socialNetwork.Label}</span>
              <ion-icon
                className="w-5 h-5 md hydrated"
                name=`logo-${socialNetwork.name}`
                role="img"
                aria-label=`logo ${socialNetwork.name}`
              ></ion-icon>
            </Link>
)})
}